openslides_backend/action/actions/organization/initial_import.py

```diff
@@ -87,6 +112,7 @@ def create_write_requests(self, instance: Dict[str, Any]) -> Iterable[WriteReque
                         fqid,
                         "initial import",
                         entry,
+                        migration_index=self.data_migration_index,
```

Comment:
Instead of giving it as an argument to the function, just create the write request first and then set the MI manually after, and then append it to the list of write requests.

Reply:
okay, will do